src/resources/postcss/tickets-admin/settings/_tickets-commerce.pcss

```diff
@@ -270,6 +270,11 @@
 	&:focus {
 		color: #fff;
 	}
+
+	&.disabled {
```

Comment:
Are we using a `.disabled` class or this should be the pseudo `:disabled`?

Reply:
the js that fetches the URL adds a `.disabled` class before the request and removes it upon completion.